pandas/core/indexes/base.py

```diff
@@ -695,13 +695,31 @@ def memory_usage(self, deep=False):
     # ops compat
     @deprecate_kwarg(old_arg_name='n', new_arg_name='repeats')
     def repeat(self, repeats, *args, **kwargs):
-        """
-        Repeat elements of an Index. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        """Repeat elements of an Index.
+
+        Parameters
+        ----------
+        repeats : int
+            The number of repetitions for each element.
+
+        Returns
+        -------
+        pandas.Index
+            Newly created Index with repeated elements.
 
         See also
         --------
         numpy.ndarray.repeat
```

Comment:
This was already like this, but now we are updating the docstring: I think it is better to change this to `numpy.repeat` (as the docstring of numpy.ndarray.repeat also refers to that one)

Reply:
Fixed!